core/prisma/schema.prisma

```diff
@@ -204,13 +204,27 @@ model PubType {
   createdAt   DateTime  @default(now())
   updatedAt   DateTime  @default(now()) @updatedAt
 
-  fields PubField[]
+  fields PubFieldToPubType[]
   pubs   Pub[]
   Form   Form[]
 
   @@map(name: "pub_types")
 }
 
+model PubFieldToPubType {
```

Comment:
this defines the join table explicitly. previously, from what I understand, prisma had made a join table behind the scenes (hence names like "A" and "B") but since we now want to add our own column, I defined it here, but tried to keep it the same as the existing table. 

this seems to work fine for kysely, but causes some trouble with some of the prisma queries